scene/2d/tile_map.compat.inc

```diff
@@ -1,5 +1,5 @@
 /**************************************************************************/
-/*  object.compat.inc                                                     */
+/*  tile_map.compat.inc                                                   */
```

Comment:
inc files are currently explicitly ignored. Should this be changed? Or are there inc files that we need to ignore?

Reply:
Traditionally `.inc` files are for data structs, which are sourced from public domain and not something we assert copyright upon, so without copyright header.

Since these new files mostly add simple proxy functions, we could remove their license headers too to keep them simple.

Reply:
We could also just not ignore `.compat.inc` while continuing to skip other `.inc` files.

Reply:
Alright, did that.